src/sentry/integrations/github/webhook.py

```diff
@@ -40,10 +40,20 @@ def __call__(self, event, host=None):
         if host:
             external_id = u'{}:{}'.format(host, event['installation']['id'])
 
-        integration = Integration.objects.get(
-            external_id=external_id,
-            provider=self.provider,
-        )
+        try:
+            integration = Integration.objects.get(
+                external_id=external_id,
+                provider=self.provider,
+            )
+        except Integration.DoesNotExist:
+            logger.info(
+                'github.missing-integration',
+                extra={
+                    'event_data': event,
```

Comment:
i think it might be a good idea to not log all of this data as it could be kind of a lot. are there some fields we can pull out that would be helpful for debugging? the action, which github org it's coming from are a couple I can think of off the top of my head.